src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/ExpectationsRepository.kt

```diff
@@ -4,5 +4,7 @@ import org.vaccineimpact.api.models.Expectations
 
 interface ExpectationsRepository : Repository
 {
-    fun getExpectationsForResponsibility(responsibilityId: Int): Expectations
+    fun getExpectationsForResponsibility(groupId: String,
```

Comment:
I changed the signature of this method since this is how the ids come in to the endpoint, and is consistent with how we do things elsewhere.